Article/Article.js

```diff
@@ -3,27 +3,41 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
+
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
+
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'Expand';
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    // console.log(this.domElement);
+    this.domElement.classList.toggle('article-open'); 
+    if (this.expandButton.textContent === 'Expand') {
```

Comment:
great job with this conditional rendering for the expand/collapse button.
you could also use a ternary statement here to do the same thing as if/else

```
expandArticle() {
  this.domElement.classList.toggle('article-open'); 
  return (this.expandButton.textContent === 'Expand') ?
   this.expandButton.textContent = 'Collapse' : 
   this.expandButton.textContent = 'Expand';
}
```